pkg/aws/amplify.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/amplify"
+	"github.com/aws/aws-sdk-go-v2/service/amplify/types"
+	"github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func NewAmplifyClient(t *testing.T, region string) *amplify.Client {
+	client, err := NewAmplifyClientE(t, region)
+	require.NoError(t, err)
+
+	return client
+}
+
+func NewAmplifyClientE(t *testing.T, region string) (*amplify.Client, error) {
+	sess, err := aws.NewAuthenticatedSession(region)
+	if err != nil {
+		return nil, err
+	}
+	return amplify.NewFromConfig(*sess), nil
+}
+
+func StartDeploymentJob(t *testing.T, client *amplify.Client, id *string, branchName *string) *string {
+	branch, err := client.GetBranch(context.Background(), &amplify.GetBranchInput{
```

Comment:
consider taking `context` as an input parameter. The caller of the function should ultimately have control over the context so they can use timeouts, etc. rather than having the context hidden within the method.

Reply:
Done